_layouts/custom/workspaces/price-plans.html

```diff
@@ -16,8 +16,7 @@ <h2 class="card-title script text-lowercase">Wayfarer</h2>
         Start with our most flexible plan and pay-as-you-go after your included hours. This plan includes your membership and 6 hours of coworking.
       </p>
       <p class="card-text">
-        <a href="/workspaces/{{ page.location | downcase | replace: " ", "-"}}/rates">see full rates</a> |
-        <a href="https://wayfare.typeform.com/to/gZ6TKU">sign up now</a>
+        <a href="/workspaces/{{ page.location | downcase | replace: " ", "-"}}/rates">see full rates</a> | <a href="{{ page.signup-url }}">sign up now</a>
```

Comment:
I actually never knew you could make page variables with dashes. I like dashes so much more. I wonder if we can go through and change them all? :) I don't like a mixture though.